libseccomp/src/notify.rs

```diff
@@ -37,30 +37,23 @@ pub type ScmpFd = RawFd;
 
 bitflags! {
     /// Userspace notification response flags
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
     pub struct ScmpNotifRespFlags: u32 {
         /// Tells the kernel to continue executing the system call that triggered the
         /// notification. Must only be used when the notification response's error and value is 0.
         const CONTINUE = SECCOMP_USER_NOTIF_FLAG_CONTINUE;
     }
 }
-impl ScmpNotifRespFlags {
-    /// Convert from underlying bit representation, preserving all bits (even those not corresponding to a defined flag).
-    // https://github.com/bitflags/bitflags/issues/263
-    #[must_use]
-    pub fn from_bits_preserve(bits: u32) -> Self {
```

Comment:
Updating bitflags to a new major version is a breaking change. Although it shouldn't have much impact since we have only one flag. Changelog is in preparation.

Correct.

Reply:
@rusty-snake Do you have any update?

Reply:
Yes, changelog is required. Will do later.